### DISCUSSION TITLE
cuOpt 23.04 release notes

### DISCUSSION THREAD

**Discussion by anandhkb** [NVIDIA/cuopt-examples#47] [Announcements]
[cuOpt-Release-Notes-23-04.pdf](https://github.com/NVIDIA/cuOpt-Resources/files/11392267/cuOpt-Release-Notes-23-04.pdf)

**Comment by damianospark** [comment]
is this release providing docker image as well?

**Reply by preethim2022** [reply]
Hi,

Thank you for your interest in cuOpt. 

cuOpt open access program was for a limited period and is no longer available for download. We will continue to share documentation to highlight updates to the solver and address any questions. 

Please reach out to cuopt@nvidia.com if you are interested in learning about licensing options and availability.

Regards,
cuOpt Team
